packages/intent-mcp/src/server.ts

```
@@ -0,0 +1,275 @@
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js'
+import {
+  IntentCoreError,
+  listIntentSkills,
+  loadIntentSkill,
+  type IntentCoreOptions,
```

ESLint

[error] Member 'IntentCoreOptions' of the import declaration should be sorted alphabetically.

(sort-imports)